projects/dotcms-field-elements/src/components/dot-tags/components/dot-autocomplete/dot-autocomplete.tsx

```diff
@@ -81,25 +87,27 @@ export class DotAutocompleteComponent {
     }
 
     private clean(): void {
-        this.getInputElement()['value'] = '';
+        this.getInputElement().value = '';
         this.cleanOptions();
     }
 
     private cleanOptions(): void {
         this.getResultList().innerHTML = '';
     }
 
-    private emitSelection(selection: string): void {
+    private emitselect(select: string): void {
         this.clean();
-        this.selection.emit(selection);
+        this.select.emit(select);
     }
 
-    private getInputElement(): HTMLElement {
+    private getInputElement(): HTMLInputElement {
         return this.el.querySelector(`#${this.id}`);
     }
 
     private initAutocomplete(): void {
-        this.autocomplete = new autoComplete({
+        this.clearList();
+        // tslint:disable-next-line:no-unused-expression
```

Comment:
why???

Reply:
Is not good to have a `new Something` and not store it anywhere, but in this case the way the autocomplete works we don't need to store the instance anywhere, we just init again when a prop change.